index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <style>
+  @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+      </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+</head>
+<body>
+    <section class="navigation">
+        <nav class="header">
+            <p class="logo">Josiah</p>
+            <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
+        </nav>
+    </section>
```

Comment:
I suggest you consider using a header tag instead of a section tag here as it is more semantic and would greatly boost SEO scores for this page. 👍 Check [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) to find out more about the header tag 😉